crates/db_views/src/private_message_view.rs

```diff
@@ -333,5 +333,14 @@ mod tests {
       .await
       .unwrap();
     assert_eq!(timmy_unread_messages, 1);
+
+    PrivateMessage::delete(pool, inserted_sara_timmy_message.id).await.unwrap();
+    PrivateMessage::delete(pool, inserted_sara_jess_message.id).await.unwrap();
+    PrivateMessage::delete(pool, inserted_timmy_sara_message.id).await.unwrap();
+    PrivateMessage::delete(pool, inserted_jess_timmy_message.id).await.unwrap();
+    Person::delete(pool, timmy.id).await.unwrap();
+    Person::delete(pool, sara.id).await.unwrap();
+    Person::delete(pool, jess.id).await.unwrap();
+    Instance::delete(pool, instance.id).await.unwrap();
```

Comment:
Due to `on delete cascade` it would be enough to call only `Instance::delete` here, right?

Reply:
Yep, that should work fine.